test/oli_web/live/community_live_test.exs

```diff
@@ -178,7 +178,7 @@ defmodule OliWeb.CommunityLiveTest do
                  name: "Testing name",
                  description: "Testing description",
                  key_contact: "Testing key contact",
-                 prohibit_global_access: true
+                 global_access: true
```

Comment:
see comment above - think you can also use params_for to pull values from factory vs hard-coded values here.